AWS_FLB_CHERRY_PICKS

```diff
@@ -92,3 +92,12 @@ https://github.com/fluent/fluent-bit.git 1.9 d3a0697c3c4a876250cc6c2c4ff44cd7790
 # windows: fix time zone offset https://github.com/fluent/fluent-bit/pull/6368
 https://github.com/fluent/fluent-bit.git 1.9 cae9a8f9cb6eab7449f14d914f9ba1d8d4e93cf0
 
+# quick fix for S3 key $INDEX bug https://github.com/aws/aws-for-fluent-bit/issues/653
```

Comment:
Do we need to merge those to Fluent Bit upstream 2.0 so that upstream customers get those fixes / enhancements?